### PR TITLE
PMM-3019 add openshift support for pmm-server docker

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

**Review by vadimtk — APPROVED** [review]

**Review by dAdAbird — APPROVED** [review]
LGTM

File: helm/helm-pmm-server/values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ imageRepo: "perconalab/pmm-server-openshift"
 imageTag: "latest"
 
 ## A chouce between "kubernetes" and "openshift"
```
Comment (dAdAbird) [inline helm/helm-pmm-server/values.yaml]: s/chouce/choice

File: helm/helm-server/values.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ nameOverride: "restore1"
 kind: "cluster"
 
 ## A chouce between "kubernetes" and "openshift"
```
Comment (dAdAbird) [inline helm/helm-server/values.yaml]: s/chouce/choice

File: helm/helm-pmm-server/values.yaml (hunk 1 above)
Comment (delgod) [reply helm/helm-pmm-server/values.yaml]: fixed

File: helm/helm-server/values.yaml (hunk 2 above)
Comment (delgod) [reply helm/helm-server/values.yaml]: fixed

**Pull request merged**